sdks/python/apache_beam/internal/gcp/auth.py

```diff
@@ -45,6 +45,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 CLIENT_SCOPES = [
+    'https://www.googleapis.com/auth/cloud-platform',
     'https://www.googleapis.com/auth/bigquery',
     'https://www.googleapis.com/auth/cloud-platform',
```

Comment:
Looks like the scope is already there, so this change isn't actually doing anything.

Reply:
Yeah, you are right. This would not have any affect.